src/core/writer.js

```diff
@@ -83,10 +83,14 @@ function writeValue(value, buffer, transform) {
     buffer.push(`(${escapeString(value)})`);
   } else if (typeof value === "number") {
     buffer.push(numberToString(value));
+  } else if (typeof value === "boolean") {
+    buffer.push(`${value.toString()}`);
```

Comment:
I might be missing something obvious here, but why do you need to wrap this in a string?
Isn't `value.toString()` *already* a string here, in which case just `buffer.push(value.toString());` ought to work.

Reply:
Also, as a follow-up, we should probably fix the pre-existing cases above to reduce unnecessary confusion when reading this code.